src/utils/constants.js

```diff
@@ -726,6 +726,9 @@ export const QUICKMARK_DERIVE_CREATE_HOLDINGS_MAPPING_ID = 'e0fbaad5-10c0-40d5-9
 export const QUICKMARK_DERIVE_CREATE_AUTHORITY_JOB_ID = '6eefa4c6-bbf7-4845-ad82-de7fc5abd0e3';
 export const QUICKMARK_DERIVE_CREATE_AUTHORITY_ACTION_ID = '7915c72e-c6af-4962-969d-403c7238b051';
 export const QUICKMARK_DERIVE_CREATE_AUTHORITY_MAPPING_ID = '6a0ec1de-68eb-4833-bdbf-0741db25c314';
+export const DEFAULT_CREATE_HOLDINGS_JOB_ID = '80898dee-449f-44dd-9c8e-37d5eb469b1d';
+export const DEFAULT_CREATE_HOLDINGS_ACTION_ID = '8aa0b850-9182-4005-8435-340b704b2a19';
```

Comment:
There's a duplicate constant above with the same id that I created before. You can delete it to remove duplicates and keep naming consistent